### PR TITLE
Add option "fixers" which allows to specify fixers which will be applied

### PR DESCRIPTION
**Pull request by aivus**
It allows to update code step-by-step instead of modifying all code in one piece.

TODO:
- [x] Update README.md

### CODE REVIEW

**Comment by aivus** [comment]
HHVM test will be green after #37

File: src/Symfony/Upgrade/Console/Command/FixCommand.php [hunk 1]
```diff
@@ -35,20 +36,29 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
             $errorsManager,
             $stopwatch
         );
-        $this->fixer->registerBuiltInFixers();
+
+        if ($fixers = $input->getOption('fixers')) {
+            $fixersList = explode(',', $fixers);
+            $fixersList = array_map('trim', $fixersList);
+            $fixer->registerBuiltInFixers($fixersList);
+
```
Comment (umpirsky) [inline src/Symfony/Upgrade/Console/Command/FixCommand.php]: Extra line.

File: src/Symfony/Upgrade/Fixer.php [hunk 2]
```diff
@@ -29,13 +30,30 @@ public function __construct(
...
+        /** @var AbstractFixer $fixer */
+        foreach (new FixerIterator() as $fixer) {
+            if (in_array($fixer->getName(), $fixerNameList, true)) {
+                $this->addFixer($fixer);
+            }
+        }
+    }
+
+    /**
+     * @param FixerInterface $fixer
```
Comment (umpirsky) [inline src/Symfony/Upgrade/Fixer.php]: This brings no value, it's already in the code, please remove.

File: src/Symfony/Upgrade/Fixer.php [hunk 3]
```diff
@@ -29,13 +30,30 @@ public function __construct(
...
+    public function registerAllBuiltInFixers()
     {
+        /** @var AbstractFixer $fixer */
         foreach (new FixerIterator() as $fixer) {
             $this->addFixer($fixer);
         }
     }
 
+    /**
+     * @param array $fixerNameList
```
Comment (umpirsky) [inline src/Symfony/Upgrade/Fixer.php]: Already in the code, brings just doubles maintenance.

File: README.md [hunk 4]
```diff
@@ -82,6 +82,11 @@ The ``--no-use-reorder`` option prevents the fixer from re-ordering USE statemen
 $ symfony-upgrade-fixer fix /path/to/code --no-use-reorder
 ```
 
+The ``--fixers`` option allows to specify which fixers should be used:
+```bash
+$ symfony-upgrade-fixer fix /path/to/code --fixers=get_request,progress_bar,property_access
```
Comment (umpirsky) [inline README.md]: Let's add some of our upgrade fixers as an example, this are PHP-CS fixers I believe.

Comment (aivus) [reply README.md]: @umpirsky I took this fixers from the next list:
https://github.com/umpirsky/Symfony-Upgrade-Fixer#fixers-available

As I understand it's a fixers from this project: https://github.com/umpirsky/Symfony-Upgrade-Fixer/tree/master/src/Symfony/Upgrade/Fixer.

But anyway, I have changed `progress_bar` on `form_events`

File: src/Symfony/Upgrade/Console/Command/FixCommand.php (hunk 1 above)
Comment (aivus) [reply src/Symfony/Upgrade/Console/Command/FixCommand.php]: done

File: src/Symfony/Upgrade/Fixer.php (hunk 3 above)
Comment (aivus) [reply src/Symfony/Upgrade/Fixer.php]: done

File: src/Symfony/Upgrade/Fixer.php (hunk 2 above)
Comment (aivus) [reply src/Symfony/Upgrade/Fixer.php]: done

File: README.md (hunk 4 above)
Comment (umpirsky) [reply README.md]: @aivus Sorry, long time ago I worked on this, already forgetting all the fixers. :)

**Pull request merged**